lib/spack/llnl/util/filesystem.py

```diff
@@ -236,25 +238,112 @@ def unset_executable_mode(path):
     os.chmod(path, mode)
 
 
-def install(src, dest):
-    """Manually install a file to a particular location."""
-    tty.debug("Installing %s to %s" % (src, dest))
+def copy(src, dst):
```

Comment:
This is picky, but I liked `dest` better than `dst` -- I think it's more readable and the rest of the code in core (except a few lines) uses `dest`.  Also changing it to `dst` makes this harder to review because it shows all those lines as changed.

Reply:
The reason I switched it from `dest` to `dst` was because all of the [`shutil`](https://docs.python.org/3/library/shutil.html) functions take `src` and `dst` as parameters. In fact, the docstrings I wrote for these functions were basically copied from the `shutil` docs.

I don't really care either way. Just let me know if you want me to change it back.

Reply:
Yeah go ahead and change it back.

Reply:
Done.